db/migrate/20230403093407_add_balance_cents_to_wallets.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class AddBalanceCentsToWallets < ActiveRecord::Migration[7.0]
+  def change
+    add_column :wallets, :balance_cents, :bigint
```

Comment:
Rails/BulkChangeTable: You can use `change_table :wallets, bulk: true` to combine alter queries. (https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table, https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html)